scripts/envs/open_manipulator/ros_interface.py

```diff
@@ -306,7 +306,6 @@ def check_for_success(self):
             self.success_count += 1
             if self.success_count == self.cfg["SUCCESS_COUNT"]:
                 print ("Current episode succeeded")
-                self.success_count = 0
```

Comment:
Why is this line deleted? To I am unfamiliar with `self.success_count` and `self.cfg["SUCCESS_COUNT"]`

Reply:
`self.cfg["SUCCESS_COUNT"]` means the number of steps stayed in boundary to goal.

Reply:
As `self.success_count` reset to 0 when `check_for_success` return `True`, I deleted it .

Reply:
I see. So, in [open_manipulator_reacher_env.py](https://github.com/kairproject/kair_algorithms_draft/blob/c1f348775e41c71d0de1bc6699f7a1732ee0bed4/scripts/envs/open_manipulator/open_manipulator_reacher_env.py#L81), if `check_for_success` is True, then `self.done` is set to true. It doesn't explicitly reset it to 0, but since the episode is complete, the next call would be `reset()`, which calls `reset_gazebo_world()` and resets the counter to 0.

Is this correct?

Reply:
Yes, that's correct.